examples/jsm/exporters/GLTFExporter.js

```diff
@@ -756,7 +756,11 @@ class GLTFWriter {
 
 			context.putImageData( composite, 0, 0 );
 
-			return new Texture( canvas );
+			const texture = new Texture( canvas );
+
+			texture.userData.mimeType = material.roughnessMap.userData.mimeType ?? material.metalnessMap.userData.mimeType ?? material.aoMap.userData.mimeType;
```

Comment:
Based on https://github.com/mrdoob/three.js/pull/23488 I think we're trying to stick to ≤ES2018 at this point, so no nullish coalescing yet.

Reply:
Ugh, okay, how's this?

Reply:
Thanks! `||` would also be fine.

Reply:
Argh, now you tell me...

Reply:
Cleaned up.